### ISSUE TITLE
Interpreting test loss as PER?

### ISSUE THREAD

**Issue by k-priyadarshi** [vrenkens/nabu#49]
HI Vincent! I have got a small problem. After training the LAS/TIMIT model and running the test returns "loss = 0.2204". Is it the Phoneme Error Rate? I am using TIMIT data and phoneme targets. Please reply.

**Comment by vrenkens** [comment]
Yes, it’s PER

**Comment by k-priyadarshi** [comment]
Thanks for a prompt reply :-)
